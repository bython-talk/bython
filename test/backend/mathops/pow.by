# RUN: %driver-full --inpath %s | FileCheck %s.stdout
def main()
{
    val powed: f32 = 2 as f32 ** 2 as i32;
    discard put_f32(powed);
}