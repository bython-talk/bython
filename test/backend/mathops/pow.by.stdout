# CHECK: 4.0